polarion/extract_from_polarion.py

```diff
@@ -0,0 +1,91 @@
+class polarian_data:
+
+    from pylero.work_item import TestCase
```

Comment:
Avoid having an import statement within implementation... it is good to have it in the top level

Reply:
This is tried and tested. When the import is kept outside of the class, the script takes lot of time to connect to polarion. Keeping the import inside seemed faster in polarion connection.